js/imageeditor/shared/utils/parse_placeholder.ts

```diff
@@ -0,0 +1,21 @@
+export function parse_placeholder(text: string): string {
+	if (!text) return "";
```

Comment:
We'll need to sanitize the the HTML first to prevent arbitrary HTML/JS from sneaking in here, right? We use DOMPurify elsewhere in the frontend

Reply:
This also seems like unnecessary weight, i don't think we need to treat this as raw html, we are only supporting a subset.